images/base/files/usr/local/bin/entrypoint

```diff
@@ -228,6 +228,27 @@ mount_kubelet_cgroup_root() {
   fi
 }
 
+# helper for cgroups v1, to eliminate the misc cgroup
+# see: https://github.com/kubernetes-sigs/kind/issues/3223
+# basically: this cgroup is not very useful for us, and until recently wasn't
+# supported by runc on cgroups v1 anyhow.
+# on cgroupsv2 we can leave it, but on v1 the mismatch in support from container
+# nesting causes problems
+remove_misc_controller() {
+  local misc_controller='/sys/fs/cgroup/misc'
+  if [[ ! -d $misc_controller ]]; then
+    return 0
+  fi
+  log_info "removing misc controller"
+  umount $misc_controller || log_warn "failed umount $misc_controller"
+  rmdir $misc_controller || log_warn "failed rmdir $misc_controller"
+  # systemd will discover misc controller is available here and re-mount it
+  # we will pretend it isn't available with a bind mount
+  # this mount will be removed by undo-mount-hacks.service
```

Comment:
I see https://github.com/kubernetes-sigs/kind/pull/3255/commits/89f3890a31353adc1ea55cad9e1372d2b0cdd5df#r1206047711

Reply:
haha, so you mount it first to avoid systemd to mount it, and later you unmount it

Reply:
yes. we mount a modified copy of the proc entry so systemd won't discover misc, then we unmount the fake. 🙃 

we also have to unmount misc because the host runtime may have already set it up.